demisto_sdk/commands/common/constants.py

```diff
@@ -496,6 +496,30 @@ def invalid_v2_script_name(file_path):
 MISC_REPUTATIONS_REGEX = r'{}{}.reputations.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
 README_REGEX = r'{}{}/readme.*\.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
 
+PACK_METADATA_NAME = 'name'
+PACK_METADATA_DESC = 'description'
+PACK_METADATA_MIN_VERSION = 'serverMinVersion'
+PACK_METADATA_CURR_VERSION = 'currentVersion'
+PACK_METADATA_AUTHOR = 'author'
+PACK_METADATA_URL = 'url'
+PACK_METADATA_EMAIL = 'email'
+PACK_METADATA_CATEGORIES = 'categories'
+PACK_METADATA_TAGS = 'tags'
```

Comment:
Also maybe should validate that tags and all keys that have list with values, aren't empty. Like we had with empty string as tags

Reply:
@IkaDemisto Are you sure that all of the list fields will be populated in a valid Packs?
what are the list fields that we know right now how to populate in every pack that exist?

Reply:
I added a validation for that, if those lists have 1 item and it's empty it will fail

Reply:
@yuvalbenshalom I meant following cases : ['tags'] =[""]
@liorblob added validation for it.